nodes/scheduling/nodes-scheduler-about.adoc

```diff
@@ -5,42 +5,47 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-
-
 Pod scheduling is an internal process that determines placement of new
 pods onto nodes within the cluster.
 
 The scheduler code has a clean separation that watches new pods
 as they get created and identifies the most suitable node to host them. It then
 creates bindings (pod to node bindings) for the pods using the master API.
 
-// The following include statements pull in the module files that comprise
-// the assembly. Include any combination of concept, procedure, or reference
-// modules required to cover the user story. You can also include other
-// assemblies.
-
 Default pod scheduling::
-{product-title} comes with a xref:../../nodes/scheduling/nodes-scheduler-default.adoc#nodes-scheduler-default[default scheduler] that serves the needs of most users. The default scheduler uses both inherent and customization tools to determine the best fit for a pod.
+{product-title} comes with a default scheduler that serves the needs of most users. The default scheduler uses both inherent and customization tools to determine the best fit for a pod.
+
+////
+// TODO: Determine if we want to pull in any info about the default scheduler from https://docs.openshift.com/container-platform/4.9/nodes/scheduling/nodes-scheduler-default.html
```

Comment:
@damemi With this PR, I'm removing this page: https://docs.openshift.com/container-platform/4.9/nodes/scheduling/nodes-scheduler-default.html

I wanted to see whether there was anything worthwhile on this page on how the default scheduler works, that should be incorporated elsewhere in the docs? For example, this explanation here:

> The default OpenShift Container Platform pod scheduler is responsible for determining placement of new pods onto nodes within the cluster. It reads data from the pod and tries to find a node that is a good fit based on configured policies. It is completely independent and exists as a standalone/pluggable solution. It does not modify the pod and just creates a binding for the pod that ties the pod to the particular node.

Or this "Understanding default scheduling" section here?

https://docs.openshift.com/container-platform/4.9/nodes/scheduling/nodes-scheduler-default.html#nodes-scheduler-default-about_nodes-scheduler-default

Reply:
@bergerhoffer yeah, I think that pulling in that general info is a good idea. That should still be documented somewhere